docker/openjdk-latest/Dockerfile

```diff
@@ -1,9 +1,9 @@
 
-FROM ubuntu:22.04
+FROM ubuntu:rolling
 ARG VERSION="dev"
 
 RUN apt-get update && \
- apt-get install --no-install-recommends -q --assume-yes openjdk-19-jre-headless=19* libjemalloc-dev=5.* && \
+ apt-get install --no-install-recommends -q --assume-yes openjdk-21-jre-headless libjemalloc-dev adduser && \
```

Comment:
just curious, what triggered the need for adduser ?

Reply:
it is no more present in the base image